### PR TITLE
Gci 1448 implement mid chd order request in chd order api

### PR DESCRIPTION
**Pull request by mouhajer-ch**
Implement Spring Boot controller in the CHD Order API method to handle the REST request from the CHD Order Consumer.

### CODE REVIEW

File: src/main/java/uk/gov/companieshouse/chd/order/api/controller/MissingImageDeliveriesController.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import uk.gov.companieshouse.chd.order.api.dto.MissingImageDeliveriesDTO;
+import uk.gov.companieshouse.chd.order.api.logging.LoggingUtils;
+import uk.gov.companieshouse.logging.Logger;
+
+@RestController
+public class MissingImageDeliveriesController {
+
+    private static final Logger LOGGER = LoggingUtils.getLogger();
+    
+    /**
```
Comment (nabeelamirch) [inline src/main/java/uk/gov/companieshouse/chd/order/api/controller/MissingImageDeliveriesController.java:26]: The spacing of this file looks off. Each indentation should be 4 spaces.

File: src/main/java/uk/gov/companieshouse/chd/order/api/controller/MissingImageDeliveriesController.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+     * Constructor.
+     */
+    public MissingImageDeliveriesController() {
+		// TODO Auto-generated constructor stub
```
Comment (nabeelamirch) [inline src/main/java/uk/gov/companieshouse/chd/order/api/controller/MissingImageDeliveriesController.java:30]: This TODO can be removed

File: src/main/java/uk/gov/companieshouse/chd/order/api/dto/MissingImageDeliveriesDTO.java [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+    @NotNull
+    @JsonProperty("payment_reference")
+    private String paymentReference;   
+
+    @NotNull
+    @JsonProperty("filing_history_category")
+    private String filingHistoryCategory;
+
+	@NotNull
```
Comment (nabeelamirch) [inline src/main/java/uk/gov/companieshouse/chd/order/api/dto/MissingImageDeliveriesDTO.java:43]: Spacing is off.

File: src/main/resources/application.properties [hunk 4]
```diff
@@ -1 +1,2 @@
-uk.gov.companieshouse.chdorderapi.health=/healthcheck
\ No newline at end of file
+uk.gov.companieshouse.chdorderapi.health=/chd-order-api/healthcheck
```
Comment (nabeelamirch) [inline src/main/resources/application.properties:1]: Good job on fixing this, another story was created to fix the issue with the healthcheck, that can be resolved.

File: src/main/java/uk/gov/companieshouse/chd/order/api/controller/MissingImageDeliveriesController.java (hunk 1 above)
Comment (mouhajer-ch) [reply src/main/java/uk/gov/companieshouse/chd/order/api/controller/MissingImageDeliveriesController.java:26]: There is any lint tool that you are using on java?

Comment (nabeelamirch) [reply src/main/java/uk/gov/companieshouse/chd/order/api/controller/MissingImageDeliveriesController.java:26]: I believe you can hookup Sonar to your IDE which can show you the issues. But I haven't done this. In intellij you can format your code by doing code > Reformat Code

Comment (mouhajer-ch) [reply src/main/java/uk/gov/companieshouse/chd/order/api/controller/MissingImageDeliveriesController.java:26]: Thanks Nabeel!

File: src/main/java/uk/gov/companieshouse/chd/order/api/controller/MissingImageDeliveriesController.java (hunk 2 above)
Comment (mouhajer-ch) [reply src/main/java/uk/gov/companieshouse/chd/order/api/controller/MissingImageDeliveriesController.java:30]: ok!

**Review by nabeelamirch — APPROVED** [review]

**Pull request merged**